e2e/random/query.go

```
@@ -105,15 +105,16 @@ func (s *QueryTestSuite) TestQueryCmd() {
+			sdk.NewCoins(sdk.NewCoin(s.Network.BondDenom, math.NewInt(10))).String(),
 		),
 	}
 
 	txResult = RequestRandomExec(s.T(), s.Network, clientCtx, from.String(), args...)
 	s.Require().Equal(expectedCode, txResult.Code)
 
-	requestID := gjson.Get(txResult.Log, "0.events.1.attributes.0.value").String()
-	requestHeight := gjson.Get(txResult.Log, "0.events.1.attributes.2.value").Int()
+	requestID := txResult.Events[8].Attributes[0].Value
+	heightStr := txResult.Events[8].Attributes[2].Value
+	requestHeight, err := strconv.ParseInt(heightStr, 10, 64)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)